4-forms-validations/form.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="es">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Form</title>
+</head>
+
+<body>
+    <form type='submit'>
+        <label for="firstName">
+            First Name:
+            <input type="text" name="firstName" id="firstName" placeholder="Ex: Juan Esteban" required>
+        </label>
+        <br>
+        <br>
+        <label for="lastName">
+            Last Name:
+            <input type="text" name="lastName" id="lastName" placeholder="Ex: Franco Estacio" required>
+        </label>
+        <br>
+        <br>
+        <label for="email">
+            Email:
+            <input type="email" name="email" id="email" placeholder="Ex: juan@gmail.com" required>
+        </label>
+        <br>
+        <br>
+        <label for="birth">
+            Birth Date:
+            <input type="date" name="birth" id="birth" placeholder="Ex: 01/01/2000" required>
+        </label>
+        <br>
+        <br>
+        <label for="gender">
+            Gender:
+            <select name="gender" id="gender" required>
+                <option value="" hidden>Seleccione...</option>
+                <option value="1">Masculino</option>
+                <option value="1">Femenino</option>
+                <option value="1">Otro</option>
+            </select>
+        </label>
+        <br>
+        <br>
+        <fieldset>
+            <legend>Status</legend>
+            <label for="1">
+                <input type="radio" name="status" id="1" value="1">
+                Estrato 1
+            </label>
```

Comment:
```suggestion
            <label for="status_1">
                <input type="radio" name="status" id="status_1" value="1">
                Estrato 1
            </label>
```
Mejor dejar explicito el valor del campo y a qué se refiere. Un id de campo `1` puede ser muy ambiguo.